build.gradle

```diff
@@ -55,7 +55,7 @@ kotlin {
 
             testTask {
                 useKarma {
-                    useChromeHeadless()
+                    useChromeHeadlessNoSandbox()
```

Comment:
Required for tests to run in container in root mode

Reply:
I just switched to Firefox, because Chrome seems not to be available on ARM64 platform.